bin/eslint-transforms.js

```diff
@@ -2,15 +2,44 @@
 
 "use strict";
 
-var nodeCLI = require("shelljs-nodecli");
+var execSync = require("child_process").execSync;
+var objectAssign = require("object-assign");
```

Comment:
If you want to target Node 4 and higher, you can just use `Object.assign` directly.